packages/aeproject-utils/utils/aeproject-utils.js

```diff
@@ -80,15 +82,24 @@ const getNetwork = (network, networkId) => {
 };
 
 const createCustomNetwork = (network, networkId) => {
-    if (network.includes('local') || networkId == undefined) {
+    if (!network || !networkId) {
         throw new Error('Both network and networkId should be passed')
     }
-    const customNetork = {
+
+    if (network === 'local' || network === 'localhost') {
```

Comment:
if the user passes localhost as his network which is different than our node (different port)? I think that localhost, should be excluded from this check